cryptofeed/rest/binance_futures.py

```diff
@@ -0,0 +1,127 @@
+'''
+Copyright (C) 2017-2020  Bryant Moscon - bmoscon@gmail.com
+
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+import hashlib
+import hmac
+import logging
+import time
+from decimal import Decimal
+from time import sleep
+
+import pandas as pd
+import requests
+from sortedcontainers import SortedDict as sd
```

Comment:
this is not being used, looks like other parts of the imports are also not used. please clean up

Reply:
I have removed the unused imports